src/styles/card.scss

```
@@ -0,0 +1,92 @@
+@import "../utils/variables";
+
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid $gray_elements;
+  background: $white_element;
+
+  &_img {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+    display: block;
+  }
+
+  &_description {
+    color: $main_accent;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    padding-bottom: 4px;
+  }
+
+  &_code {
+    color: $secondary;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    padding-bottom: 16px;
+  }
+
+  &_review {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    width: 166px;
+    padding-bottom: 24px;
+
+    &_count {
+      display: flex;
+      align-items: center;
+      color: $main_accent;
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+    }
+  }
+
+  &_price {
+    display: flex;
+    justify-content: space-between;
+    padding-bottom: 16px;
+
+    &_text {
+      color: $secondary;
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+    }
+
+    &_amount {
+      color: $main_accent;
+      text-align: right;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+    }
+  }
+
+  &_button-buy {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 40px;
+    width: 166px;
```

```suggestion
    width: 100%;
```
do not hardcode button's width. make it on full width of card